README.md

```diff
@@ -284,3 +285,21 @@ given our lack of previous experience in such a scenario. For this we are proud.
     - Meeting moderation and adgenda creation
     - Author of the PUM function (code review by Anders)
 
+## Building
+
+For building and running this project the requirements are:
+
+- [Maven v3.8.6](https://maven.apache.org/docs/3.8.6/release-notes.html)
+- Java 19+
+
+### How to build and run
+
+1. Clone this repository.
+2. Import project as a Maven project with your desired IDE.
```

Comment:
describe what is meant my 'a Maven project' or explain how to do it / how it differs from just opening a folder in the IDE like normal

Reply:
I mean, thought an IDE you either choose "gradle", "maven", or anything else. I don't know what else to write.